### ISSUE TITLE
[Link and button labels] Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s. (08.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61936]
#### VFS team name
10-10 Health Apps

#### VFS product name
10-10EZ

#### VFS feature name
Household Information Optimization

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s.

### Issue Details
Language used in form questions and URL segments is not consistent across this product. Pages in question include: 1.) the financial onboarding page and 2.) all pages in Step 4 regarding personal information. 
* The financial onboarding page has a URL segment "/financial-onboarding" which is jargon and reflects internal organization and does not follow design system [content guidelines for URLs](https://design.va.gov/content-style-guide/url-standards).
   * The term "onboarding" also does not appear in the page title. Page titles and URLs should be close matches and should not introduce new language. 
* The personal information sections in Step 4 have inconsistent terminology and URLs compared with the earlier personal information page in Step 1: Veteran Information section. These should be consistent whenever possible. Please see differentiation in this bulleted list:
   * Veteran information step 1: "Veteran's Personal Information" (matching url: /personal-information)
   * Spouse information step 4: "Spouse Information" (url not quite a match: /spouse-basic-information)
   * Dependent information step 4: "Dependent's Basic Information (url not quite a match: /dependent-information)

### VA.gov Experience Standard
[Category Number 08, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.6_AA

### Platform Recommendation
Please work with Sitewide CAIA for support in assigning consistent terminology across page titles/form questions and URLs, even if these URLs are currently live. They can be really helpful with fixing these over time :-) @jennymayoco @DanielleThierryUSDSVA 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by hdjustice** [comment]
The following changes will be made for this ticket:
- Initial Financial Info page URL will be updated to `/financial-information-use`
- Share Page (do you want to share) URL will be updated to `/share-financial-information`
- Confirmation Page (confirmation to not share) URL will be updated to `/share-financial-information-confirm`
- Info Needed Page (What financial info needed) URL will be updated to `/financial-information-needed`
- Initial Spouse Info Page URL will be updated to `/spouse-personal-information`
- Initial Spouse Info Page (name, dob, ssn, etc) title will be updated to "Spouse's personal information"
- Initial Dependent Info Page (name, dob, ssn, etc) title will be updated to "Dependent's personal information"

cc: @longmd @jessicastump

**Comment by longmd** [comment]
@sterkenburgsara , these have been updated per Heather's note above and are ready for review and final sign-off.

**Comment by sterkenburgsara** [comment]
Hi Matt - thank you so much for the update! This looks great to me. For
your awareness - after the staging review (unless you have questions or
want follow up) you aren’t required to run anything by governance team
folks. We trust you to close the issues once you think they’ve been
resolved. You are good to go - wishing you a happy launch y’all! Wahoo! 😊

**Comment by hdjustice** [comment]
Validated changes made as per our updated design decisions.

- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/0ea86e69-e2ca-47ab-bc84-e8de7a249df9)

- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/2245c339-dabf-45db-9671-f7b4d1cb8527)

- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/2e7bdeaf-5472-42cd-b261-4384f572055b)

- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/96e312cd-3dcb-49d2-9400-8572ab896d9a)



- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/2cdf2f6f-bd37-467f-b3c9-226e955f87d4)

- ![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/92328831/8f74a5e3-cba4-45c9-8801-ecbcea8cee50)

-
